socialhome/utils.py

```diff
@@ -0,0 +1,7 @@
+from django.conf import settings
+
+
+def get_full_media_url(path):
+    return "{url}{media}{path}".format(
+        url=settings.SOCIALHOME_URL, media=settings.MEDIA_URL, path=path,
```

Comment:
Would love to but need to support Python 3.4 and 3.5 for a while still. Ubuntu, which is easily one of the most popular installation servers, ships 3.4 in 14.04 which is still supported.